internal/ext/importer.go

```diff
@@ -113,12 +115,30 @@ func (i *Importer) Import(ctx context.Context, enc Encoding, r io.Reader) (err e
 			createdSegments = make(map[string]*flipt.Segment)
 			// map flagKey:variantKey => *variant
 			createdVariants = make(map[string]*flipt.Variant)
+			// map flagKey => bool
+			existingFlags = make(map[string]bool)
+			// map segmentKey => bool
+			existingSegments = make(map[string]bool)
 		)
 
+		if skipExisting {
+			existingFlags, err = i.existingFlags(ctx, namespace)
+			if err != nil {
+				return fmt.Errorf("getting existing flags: %w", err)
+			}
+
+			existingSegments, err = i.existingSegments(ctx, namespace)
+			if err != nil {
+				return fmt.Errorf("getting existing flags: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("getting existing segments: %w", err)
```

Reply:
actually we could probably just do `return err` here and above, since the error you are wrapping is already wrapped in the function you're calling.

currently, it will be like:

```
getting existing segments: listing segments: {err}
```

but if we just return `err` here it will be:

```
listing segments: {err}
```

Reply:
Fixed, thanks